src/applications/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -27,6 +27,18 @@ export const APPEAL_TYPES = {
 
 export const appealTypes = Object.values(APPEAL_TYPES);
 
+export const programAreaMap = {
```

Comment:
@Mottie I needed this map as well so I exported it from here

Reply:
Sounds good! Would it be better to move this to the `vets-json-schema` constants?

Reply:
No worries... it was just a thought.